msgcntr/messageforums-app/src/webapp/js/sak-10625.js

```diff
@@ -2,8 +2,12 @@
 var click=0;
 sak10625_disabler = function(){
   var existing_event = this.onclick;
+  $("a, input[type=button], input[type=submit]").attr("onclick", "");
```

Comment:
I just tried
  $("a, input[type=button], input[type=submit]").off("click")

In replace of this line and it doesn't seem to be doing anything. Inspecting the source in the previous case the onclick attribute is actually removed, with the .off it's still there. Either it's the wrong syntax, it only works in combination with .on() or it doesn't do the same thing.